src/main/resources/org/sagebionetworks/web/client/widget/footer/FooterViewImpl.ui.xml

```diff
@@ -39,7 +39,7 @@
 							<bh:Div width="100%" addStyleNames="flexcontainer-row">
 								<g:Image ui:field="alphaModeStatusIcon" addStyleNames="flexcontainer-column margin-right-5 margin-top-3" width="18px" height="18px" />
 								<b:Anchor ui:field="debugLink"
-									addStyleNames="flexcontainer-column flexcontainer-justify-flex-end color-white">
+									addStyleNames="flexcontainer-column flexcontainer-justify-flex-end color-white font-size-14">
```

Comment:
Fix for SWC-5945: revert size of Experimental Mode ON/OFF text (back to font-size 14px)